ethcore/verification/src/verification.rs

```diff
@@ -650,10 +650,6 @@ mod tests {
 		let mut bad_header = good.clone();
 		bad_header.set_transactions_root(eip86_transactions_root.clone());
 		bad_header.set_uncles_hash(good_uncles_hash.clone());
-		match basic_test(&create_test_block_with_data(&bad_header, &eip86_transactions, &good_uncles), engine) {
-			Err(Error::Transaction(ref e)) if e == &parity_crypto::publickey::Error::InvalidSignature.into() => (),
-			e => panic!("Block verification failed.\nExpected: Transaction Error (Invalid Signature)\nGot: {:?}", e),
-		}
```

Comment:
this was required (as some other removed code) to account for eip86 which we no longer support